docs/UserGuide.md

```diff
@@ -3,39 +3,46 @@ layout: page
 title: User Guide
 ---
 
-FitBook is a **desktop app for managing clients, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you are a personal trainer that prefers typing over mouse interactions, FitBook is the perfect application for you to manage your clients!
+FitBook is a **desktop app for managing clients, optimized for use via a Command Line Interface** (CLI) while still 
+having the benefits of a Graphical User Interface (GUI). If you are a personal trainer that prefers typing over mouse 
+interactions, FitBook is the perfect application for you to manage your clients!
 
 * Table of Contents
 {:toc}
 
---------------------------------------------------------------------------------------------------------------------
-
-## Quick start
-
-1. Ensure you have Java `11` or above installed in your Computer.
+<hr>
 
-1. Download the latest `FitBook.jar` from [here](https://github.com/AY2324S2-CS2103T-T17-3/tp/releases).
+## Quick Start
 
-1. Copy the file to the folder you want to use as the _home folder_ for your address book.
+1. Ensure you have Java `11` or above installed.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar FitBook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+2. Download the latest version of `FitBook.jar` from [here](https://github.com/AY2324S2-CS2103T-T17-3/tp/releases).
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
+3. Copy the file to the folder you want to use as the **home folder** for FitBook.
 
-   * `list` : Lists all contacts.
+4. Open a command terminal, `cd` into the folder with FitBook.jar, and run the `java -jar FitBook.jar` command 
```

Comment:
The default method of launching FitBook should probably be double clicking on the jar file. Since we are under the presumption that they have Java `11` already installed, it should already be associated with the file type.

Reply:
okay have made the relevant changes @jjchee77